SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1332,6 +1335,55 @@ void USpatialPendingNetGame::SendJoin()
 	bSentJoinRequest = true;
 }
 
+Worker_EntityId USpatialNetDriver::SetupActorEntity(AActor* Actor)
+{
+	checkSlow(Actor);
```

Comment:
I think checkSlows are generally used when the check is expensive and should only run during debug since performance isn't a concern then. In this case I would still use a check.